control-plane/csi-driver/src/bin/controller/controller.rs

```diff
@@ -402,6 +400,9 @@ impl rpc::csi::controller_server::Controller for CsiControllerSvc {
 
         let params = PublishParams::try_from(&args.volume_context)?;
 
+        // Extract the change fs id from the params if present.
+        let change_fs_id = *params.change_fs_id();
```

Comment:
hmm should we process this at all for the moment?

Reply:
Shall I return an error here if this is Not None?

Reply:
Or just make it false even if its true for now?